ui/lib/kv/addon/components/page/secret/edit.js

```diff
@@ -34,7 +34,7 @@ export default class KvSecretEdit extends Component {
   @tracked modelValidations;
   @tracked invalidFormAlert;
 
-  get showAlert() {
+  get showOldVersionAlert() {
     const { currentVersion, previousVersion } = this.args;
```

Comment:
since previousVersion is no longer passed in this _should_ be breaking a test to confirm the banner renders 😅

Reply:
Nope, I didn't change it on this component!

Reply:
Yep! Sorry about my confusion there 🙈